specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -2685,7 +2712,7 @@
       },
       "description": "The List WorkspaceQuotasByVMFamily operation response."
     },
-    "Identity": {
+    "AMLIdentity": {
```

Comment:
>AMLIdentity [](start = 5, length = 11)

I see quite a few changes like this. Nothing's wrong with it technically, but some of these contracts ("Resource", "Identity", etc.) are not really specific to AML.

Reply:
The body of the definitions here are different from the types in types.json such as declaring UserAssigned for Identities. These cause conflicts between types.json and machineLearningServices.json, since the definitions have the same name.

Reply:
You can redeclare definitions with the same name in your own swaggers, but it's ok as-is - was just unusual.

---
In reply to: [496870760](https://github.com/Azure/azure-rest-api-specs/pull/10926#discussion_r496870760) [](ancestors = 496870760)

Reply:
Actually nevermind. Some of those changes are being flagged as breaking. Let's discuss over the email thread.

---
In reply to: [497156386](https://github.com/Azure/azure-rest-api-specs/pull/10926#discussion_r497156386) [](ancestors = 497156386,496870760)